### ISSUE TITLE
Define my own table structure

### ISSUE THREAD

**Issue by jason1378456831** [MadcowD/ell#295]
When calling LMP, a database file ell.db is automatically created in the logdir directory, I want to create my own defined tables and fields when initializing the table building, what do I need to do?

**Comment by MadcowD** [comment]
Hmm can you elaborate or come on the discord to chat in general? What do you want to add?

**Comment by jason1378456831** [comment]
> Hmm can you elaborate or come on the discord to chat in general? What do you want to add?

For example, after I start ell-studio, the default window that opens shows multiple called LMP nodes (which call and collaborate with each other via prompt that have been encapsulated), which I could string together into a flowchart, but this flowchain may only apply to one scene.
I have more LMPs to help me accomplish more in order to create multiple scenes that are isolated from each other and do not interfere with each other. Based on this requirement, I would like to create multiple secondary labels under “Model” to differentiate the LMP process chain in different scenarios. For example, Scenario A has 5 LMPs, which can help me accomplish one thing in a chain; Scenario B has 10 LMPs, which can help me accomplish another thing in a chain.
So my idea is, when creating ell.db, what about the initialized tables and the table structure fields, e.g. I would like to generate other custom tables panel, fields panel_id, panel_name, etc. at initialization in addition to the five tables already given, so as to achieve the purpose of my secondary development?
In a nutshell, what I want to know is which or what are the files in the project that are involved in building the initialization of the tables so that I can generate my custom tables by default when initializing the database files?
